planning/motion_velocity_smoother/include/motion_velocity_smoother/resample.hpp

```diff
@@ -43,12 +43,12 @@ struct ResampleParam
   double sparse_min_interval_distance;  // minimum points-interval length for sparse sampling [m]
 };
 
-boost::optional<TrajectoryPoints> resampleTrajectory(
+TrajectoryPoints resampleTrajectory(
```

Comment:
@TakaHoribe I think it's better to remove unnecessary header(`boost/optional.hpp`).

Reply:
Thanks! I'll take it